test/http2_spec_5_1_SUITE.erl

```diff
@@ -212,53 +212,60 @@ half_closed_remote_sends_headers(_Config) ->
         ],
 
     {H1, EC} =
-        http2_frame_headers:to_frames(1,
-                                      RequestHeaders,
-                                      hpack:new_context(),
-                                      16384,
-                                      true),
+        h2_frame_headers:to_frames(1,
+                                   RequestHeaders,
+                                   hpack:new_context(),
+                                   16384,
+                                   true),
 
     http2c:send_unaltered_frames(Client, H1),
 
     %% The stream should be half closed remote now
 
     {H2, _EC2} =
-        http2_frame_headers:to_frames(1,
-                                      RequestHeaders,
-                                      EC,
-                                      16384,
-                                      true),
+        h2_frame_headers:to_frames(1,
+                                   RequestHeaders,
+                                   EC,
+                                   16384,
+                                   true),
 
 
     http2c:send_unaltered_frames(Client, H2),
-
-    Resp = http2c:wait_for_n_frames(Client, 1, 3),
+     Resp = http2c:wait_for_n_frames(Client, 1, 3),
     ct:pal("Resp: ~p", [Resp]),
-    ?assertEqual(true, length(Resp) >= 3),
-    [ {HeadersH, _}, {DataH, _}, {RstStreamH, RstStream}|_] = Resp,
-    ?assertEqual(?HEADERS, HeadersH#frame_header.type),
-    ?assertEqual(?DATA, DataH#frame_header.type),
-    ?assertEqual(?RST_STREAM, RstStreamH#frame_header.type),
-    ?assertEqual(?STREAM_CLOSED, http2_frame_rst_stream:error_code(RstStream)),
+    ?assertEqual(true, (length(Resp) >= 1)),
+
+    %% We need to find if one of these things are a RstStream
+    RstStreams =
```

Comment:
Nope! That's why I made it a separate commit *AND* told you "hey, the only thing really interesting here is this commit" :P